MicrophonePolly.html

```diff
@@ -145,6 +108,33 @@ <h5>* Will use browser's default audio device if there is no selection</h5>
 			</div>
 		</div>
 
+		<div class="ui divider"></div>
```

Comment:
I'll look into throwing my section into an accordion. How would you feel about moving the disclaimer section to the top?

Reply:
![accordion](https://user-images.githubusercontent.com/86703073/217895036-49c9b5b1-1fd7-45d4-abf2-2f01bcdfd39b.PNG)
I've added an accordion. 

I've been also toying around around with other apis to provide TTS. I think I can get tts monster and uberduck both working but they are both paid and I'm not sure how much interested there is in getting them in. If you want to move to including these, we'd probably need to rework the ui so that the disclaimer as well as login ui are consolidated per service.

Reply:
Another option that we might consider is using streamelements as a provider since they use polly under the covers. Only issue with this is having to hardcode the voice lists.

I made a streamelements TTS widget a while back. The user would have to get their api key though: https://github.com/swolekat/se-widgets/blob/main/tts/tts.js#L13